### PR TITLE
[Mandiant] Add log when exception is caught outside of running connector process

### PR DESCRIPTION
**Pull request by helene-nguyen**
### Proposed changes

* Catch exception outside running connector process by adding a print()

### Related issues

* https://github.com/OpenCTI-Platform/connectors/issues/2372

### Checklist

- [x] I consider the submitted work as finished
- [ ] I tested the code for its functionality using different use cases
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### CODE REVIEW

File: external-import/mandiant/src/connector/__main__.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
     mandiantConnector = Mandiant()
     while True:
         mandiantConnector.run()
-except Exception:
+except Exception as err:
+    print(err)
     time.sleep(10)
     sys.exit(0)
```
Comment (gileri) [inline external-import/mandiant/src/connector/__main__.py]: Hi, shouldn't we either raise/not catch the exception in order to exit with a non-zero status code ?

That will help monitor critical errors of this connector leading to unexpected shutdowns.

File: external-import/mandiant/src/connector/__main__.py [hunk 2]
```diff
@@ -7,6 +7,7 @@
     mandiantConnector = Mandiant()
     while True:
         mandiantConnector.run()
-except Exception:
+except Exception as err:
+    print(err)
     time.sleep(10)
```
Comment (gileri) [inline external-import/mandiant/src/connector/__main__.py]: Not a big deal and mostly out-of-scope, but that sounds like this shouldn't be handled inside the connector, but on what is managing its process (systemd, supervisord etc.).

A lot of connectors have this.

Comment (helene-nguyen) [reply external-import/mandiant/src/connector/__main__.py]: This is a little fix for this connector. 
I go along with you @gileri that we may have another process to catch the exception.
Let's talk about it :)

**Review by Megafredo — APPROVED** [review]

File: external-import/mandiant/src/connector/__main__.py (hunk 2 above)
Comment (helene-nguyen) [reply external-import/mandiant/src/connector/__main__.py]: @gileri Following our discussion, we will add a track trace to better log exception :)

**Pull request merged**